README.md

```diff
@@ -103,4 +103,13 @@ In order to produce a uniquely identifiable distribution:
    the [``build/number``](http://conda.pydata.org/docs/building/meta-yaml.html#build-number-and-string).
  * If the version of a package **is** being increased, please remember to return
    the [``build/number``](http://conda.pydata.org/docs/building/meta-yaml.html#build-number-and-string)
-   back to 0.
\ No newline at end of file
+   back to 0.
+
+Feedstock Maintainers
+=====================
+
+* [@FlanFlanagan](https://github.com/FlanFlanagan/)
+* [@bam241](https://github.com/bam241/)
+* [@gonuke](https://github.com/gonuke/)
+* [@scopatz](https://github.com/scopatz/)
+* [@katyhuff](https://github.com/katyhuff/)
```

Comment:
Did you do this manually?  This gets generated by rerendering.

Reply:
yes - just pushed a revert of this